### ISSUE TITLE
Stalls when processing test bodies

### ISSUE THREAD

**Issue by mjdease** [adamgruber/mochawesome#257]
**Describe the bug**
Test runs appear to stall at the end when using this reporter.
I'm upgrading from v3.0.2 to 3.1.0, it appears this new regex used in `cleanCode` chokes on certain strings:
https://github.com/adamgruber/mochawesome/blob/e176d7fd3fce7067d125ee7f7fe13dbfc0fa4f7d/src/utils.js#L56

**Code Reproduce**
I pulled out the regex from `cleanCode` and used a sample string to demonstrate the issue. Run the following lines in a node session and notice the `str.replace` call does not complete.
```node
> let str =  "\"Failure type was error - return eventSpies.validationError.dispatch.lastCall.args[0].errorResponse.type === 'error';\"\
\"Failure code was service error - return eventSpies.validationError.dispatch.lastCall.args[0].errorResponse.code === 4;\
\"Blackout validator failed - return eventSpies.validationError.dispatch.lastCall.args[0].errorResponse.validatorType === 'top.validator.blackout';\"\
\"Validation failed once - return eventSpies.validationError.dispatch.calledOnce;\"\
\"Content failed from error  - return eventSpies.contentError.dispatch.calledOnce;\"\
\"Content state is error - return player.model().contentState() === TOP.player.ContentState.Error;\""

> str = str.replace(/\r\n|[\r\n\u2028\u2029]/g, '\n').replace(/^\uFEFF/, '').replace(/^(?:.|\s)*?(?:{|=>) *\n?(?:\(|{)?/, '')
```

**Expected behavior**
String processing should complete.

**Environment (please complete the following information):**
 - Platform macOS 10.14
 - mochawesome version 3.1.0
 - mocha (or other test library) 5.2.0
 - Node version 10.12.0 (also observed on 9.5.0)

**Additional context**
Our test implementation overrides the test bodies with formatted versions of our assertions which is why the sample string used in the example doesn't look like a valid function.

**Comment by adamgruber** [comment]
Fixed in 3.1.1

**Comment by mjdease** [comment]
Thanks for the quick fix!
